File: BridgeX/app/src/main/java/cz/lriedel/bridgex/CustomWebChromeClient.java
package cz.lriedel.bridgex;

import android.webkit.GeolocationPermissions;
import android.webkit.WebChromeClient;

public class CustomWebChromeClient extends WebChromeClient {

    @Override
    public void onGeolocationPermissionsShowPrompt(String origin, GeolocationPermissions.Callback callback) {
        callback.invoke(origin, true, false);
    }
}

File: BridgeX/app/src/main/java/cz/lriedel/bridgex/ui/CustomWebChromeClient.kt
package cz.lriedel.bridgex.ui

import android.webkit.GeolocationPermissions
import android.webkit.WebChromeClient

class CustomWebChromeClient : WebChromeClient() {
    override fun onGeolocationPermissionsShowPrompt(origin: String, callback: GeolocationPermissions.Callback) {
        callback.invoke(origin, true, false)
    }
}
